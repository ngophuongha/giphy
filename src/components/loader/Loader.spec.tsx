import { render } from "../../../test";
import { Loader, Suspender } from "./index";

describe("Loader", () => {
  it("should match snapshot", () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});

describe("Suspender", () => {
  it("should match snapshot", () => {
    const { container } = render(<Suspender />);
    expect(container).toMatchSnapshot();
  });
});
