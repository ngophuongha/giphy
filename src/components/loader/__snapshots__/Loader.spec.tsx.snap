// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot 1`] = `
<div>
  <div
    class="g-2 row row-cols-xl-4 row-cols-md-3 row-cols-2"
  >
    <div
      class="col"
    >
      <div
        class="mt-2 card"
      >
        <img
          class="card-img-top"
          src="placeholder-img.png"
        />
        <div
          class="card-body"
        >
          <div
            class="placeholder-glow card-title h5"
          >
            <span
              class="col-6 placeholder"
            />
          </div>
          <p
            class="placeholder-glow card-text"
          >
            <span
              class="col-7 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-6 placeholder"
            />
             
            <span
              class="col-8 placeholder"
            />
          </p>
        </div>
      </div>
    </div>
    <div
      class="col"
    >
      <div
        class="mt-2 card"
      >
        <img
          class="card-img-top"
          src="placeholder-img.png"
        />
        <div
          class="card-body"
        >
          <div
            class="placeholder-glow card-title h5"
          >
            <span
              class="col-6 placeholder"
            />
          </div>
          <p
            class="placeholder-glow card-text"
          >
            <span
              class="col-7 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-6 placeholder"
            />
             
            <span
              class="col-8 placeholder"
            />
          </p>
        </div>
      </div>
    </div>
    <div
      class="col"
    >
      <div
        class="mt-2 card"
      >
        <img
          class="card-img-top"
          src="placeholder-img.png"
        />
        <div
          class="card-body"
        >
          <div
            class="placeholder-glow card-title h5"
          >
            <span
              class="col-6 placeholder"
            />
          </div>
          <p
            class="placeholder-glow card-text"
          >
            <span
              class="col-7 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-4 placeholder"
            />
             
            <span
              class="col-6 placeholder"
            />
             
            <span
              class="col-8 placeholder"
            />
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Suspender should match snapshot 1`] = `
.c0 {
  background-color: #fff;
  color: #000000;
  overflow-x: hidden;
}

<div>
  <div
    class="c0"
  >
    <div
      class="spinner-grow"
    />
  </div>
</div>
`;
